### DISCUSSION TITLE
Middleware Bug

### DISCUSSION THREAD

**Discussion by coder-abhi27** [vercel/next.js#57710] [Help]
### Summary

Hi Team, actually there is a bug https://github.com/vercel/next.js/issues/57587. It is regarding some middleware bug. I am actually new to codebase. I want to contribute. 

I wanted help in can someone tell which part of the code this issue is related and any guidance to solve this.

### Example

https://github.com/vercel/next.js/issues/57587

**Comment by aziyatali** [comment]
You can refer to this [link](https://nextjs.org/docs/app/building-your-application/caching#opting-out-2) for the issue. As I believe issue is coming from there.
If you want to contribute, you can simply open PR with your proposition.
